pepper-apis/studybuilder-cli/studies/osteo/study-events.conf

```diff
@@ -493,6 +493,178 @@
       "dispatchToHousekeeping": false,
       "order": 1
     },
+    ##CONSENT to ABOUTYOU if RELEASE is submitted.. handle osteo2 edge case
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "ABOUTYOU"
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["ABOUTYOU"].hasInstance() ||
+       (user.studies["CMI-OSTEO"].forms["CONSENT_ADDENDUM"].hasInstance() && user.studies["CMI-OSTEO"].forms["CONSENT_ADDENDUM"].isStatus("CREATED", "IN_PROGRESS")) ||
+       (user.studies["CMI-OSTEO"].forms["RELEASE_SELF"].hasInstance() && user.studies["CMI-OSTEO"].forms["RELEASE_SELF"].isStatus("CREATED", "IN_PROGRESS"))""",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 6
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "ABOUT_YOU_ACTIVITY"
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT_ADDENDUM"].hasInstance() && user.studies["CMI-OSTEO"].forms["CONSENT_ADDENDUM"].isStatus("CREATED", "IN_PROGRESS")""",
+      "preconditionExpr": """user.studies["CMI-OSTEO"].forms["ABOUTYOU"].hasInstance() && user.studies["CMI-OSTEO"].forms["ABOUTYOU"].isStatus("COMPLETE"))"""
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 7
+    },
+
+    ##PARENTAL_CONSENT to ABOUTYOU if RELEASE_MINOR is submitted.. handle osteo2 edge case
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "PARENTAL_CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "ABOUTYOU"
```

Comment:
If we did PARENTAL_CONSENT, isn't it pediatric and we should create "ABOUTCHILD" ?